### ISSUE TITLE
anime-downloader formula test issue blocks CI

### ISSUE THREAD

**Issue by chenrui333** [Homebrew/homebrew-core#49339]
Relates to 
- https://github.com/Homebrew/homebrew-core/pull/49255
- https://github.com/Homebrew/homebrew-core/pull/49306

**Comment by chenrui333** [comment]
Manually tried installed and tested couple of providers:

```
$ /usr/local/Cellar/anime-downloader/4.0.1/bin/anime watch -n Kaguya-sama --provider 9anime
anime: anime-downloader 4.0.1
ERROR: 9anime introduced captcha. You can use `anime -ll DEBUG` to use it.

$ /usr/local/Cellar/anime-downloader/4.0.1/bin/anime watch -n Kaguya-sama --provider twist.moe
anime: anime-downloader 4.0.1
ERROR: 403 Client Error: Forbidden for url: https://twist.moe/

$ /usr/local/Cellar/anime-downloader/4.0.1/bin/anime watch -n Kaguya-sama --provider masterani
anime: anime-downloader 4.0.1
ERROR: 'NoneType' object has no attribute 'search'
```

**Comment by samford** [comment]
I tested this as well and wasn't able to get any of the providers to work.  Looking at the anime-downloader repo's issues, it's not uncommon for issues to arise with providers.  The current test interacts with providers, which is why it's failing now and will do so again in the future as providers come and go.

Understanding this instability, it's probably best to limit the anime-downloader test to simply checking the version, as that appears to be the only way to test the executable without interacting with a provider.  With this in mind, I've opened a PR (#49696) that reduces the test to a version check.
